### PR TITLE
Add File::getStream() method

### PR DESCRIPTION
**Pull request by DarkSide666**

### CODE REVIEW

File: src/Model/File.php [hunk 1]
```diff
@@ -264,6 +266,17 @@ public function createThumbnail(string $path): bool
         return true;
     }
 
+    public function getStream(): StreamInterface
+    {
+        $this->assertIsEntity();
+
```
Comment (mvorisek) [inline src/Model/File.php]:
```suggestion
```

Comment (mvorisek) [reply src/Model/File.php]: `get` below is allowed on entity only

File: src/Model/File.php [hunk 2]
```diff
@@ -264,6 +266,17 @@ public function createThumbnail(string $path): bool
...
+        $path = $this->get('location');
+        $resource = $this->flysystem->readStream($path);
+        $stream = (new Psr17Factory())->createStreamFromResource($resource);
```
Comment (mvorisek) [inline src/Model/File.php:275]: IFAK this is used in this rpeo, so dedup

File: src/Helper.php [hunk 3]
```diff
@@ -15,11 +14,7 @@ class Helper
      */
     protected static function terminate(File $model, App $app): void
```
Comment (mvorisek) [inline src/Helper.php]: rename `$model` to `$entity`

**Comment by DarkSide666** [comment]
Review comments addressed

File: src/Model/File.php [hunk 4]
```diff
@@ -247,7 +254,7 @@ public function createThumbnail(string $path): bool
             $uri = stream_get_meta_data($thumbFile)['uri'];
 
             // save thumbnail
-            $thumbName = basename($this->get('meta_filename'), $this->thumbnailFormat) . '.thumb' . $this->thumbnailFormat;
+            $thumbName = basename($this->get('meta_filename')) . '.thumb' . $this->thumbnailFormat;
```
Comment (mvorisek) [inline src/Model/File.php]:
```suggestion
            $thumbName = basename($this->get('meta_filename')) . '.thumb.' . $this->thumbnailFormat;
```

**Pull request merged**